components/multi-platform-controller/production/kflux-prd-rh02/host-config.yaml

```diff
@@ -437,36 +438,20 @@ data:
   dynamic.linux-ppc64le.max-instances: "30"
   dynamic.linux-ppc64le.allocation-timeout: "1800"
 
-# Power instances static VM
-  # host.power-rhtap-prod-1.address: "169.47.123.234"
-  # host.power-rhtap-prod-1.platform: "linux/ppc64le"
-  # host.power-rhtap-prod-1.user: "root"
-  # host.power-rhtap-prod-1.secret: "ibm-production-ppc64le-ssh-key"
-  # host.power-rhtap-prod-1.concurrency: "2"
-
-  # # host.power-rhtap-prod-2.address: "169.47.123.236"
-  # # host.power-rhtap-prod-2.platform: "linux/ppc64le"
-  # # host.power-rhtap-prod-2.user: "root"
-  # # host.power-rhtap-prod-2.secret: "ibm-production-ppc64le-ssh-key"
-  # # host.power-rhtap-prod-2.concurrency: "2"
-
-  # host.power-rhtap-prod-3.address: "169.47.123.238"
-  # host.power-rhtap-prod-3.platform: "linux/ppc64le"
-  # host.power-rhtap-prod-3.user: "root"
-  # host.power-rhtap-prod-3.secret: "ibm-production-ppc64le-ssh-key"
-  # host.power-rhtap-prod-3.concurrency: "2"
-
-  # host.power-rhtap-prod-4.address: "150.239.181.173"
-  # host.power-rhtap-prod-4.platform: "linux/ppc64le"
-  # host.power-rhtap-prod-4.user: "root"
-  # host.power-rhtap-prod-4.secret: "ibm-production-ppc64le-ssh-key"
-  # host.power-rhtap-prod-4.concurrency: "2"
-
-  # host.power-rhtap-prod-5.address: "150.239.181.172"
-  # host.power-rhtap-prod-5.platform: "linux/ppc64le"
-  # host.power-rhtap-prod-5.user: "root"
-  # host.power-rhtap-prod-5.secret: "ibm-production-ppc64le-ssh-key"
-  # host.power-rhtap-prod-5.concurrency: "2"
+# PPC64LE large nodes
+  dynamic.linux-large-ppc64le.type: ibmp
+  dynamic.linux-large-ppc64le.ssh-secret: "ibm-production-ppc64le-ssh-key"
+  dynamic.linux-large-ppc64le.secret: "ibm-api-key"
+  dynamic.linux-large-ppc64le.key: "prod-konflux-infra-external"
+  dynamic.linux-large-ppc64le.image: "kflux-ppc-base-04-feb-2025"
+  dynamic.linux-large-ppc64le.crn: "crn:v1:bluemix:public:power-iaas:dal12:a/8cfd5baa22024ab9822f1e9d4659a5e5:e08f6cb5-9d8c-4931-87c2-16580c2ffe0a::"
+  dynamic.linux-large-ppc64le.url: "https://us-south.power-iaas.cloud.ibm.com"
+  dynamic.linux-large-ppc64le.network: "54fc8e49-fcb0-4842-9802-405865f0597d"
+  dynamic.linux-large-ppc64le.system: "e980"
+  dynamic.linux-large-ppc64le.cores: "4"
+  dynamic.linux-large-ppc64le.memory: "16"
+  dynamic.linux-large-ppc64le.max-instances: "10"
```

Comment:
PLease set this to 30 so far (due to bug it actually counts all machines, not per-flavour, we will fix that soon)